xsd/pointwise_models/gard.py

```diff
@@ -92,8 +92,8 @@ def _predict_one_step(self, X):
         _, inds = self.kdtree_.query(X, k=self.n_analogs, **self.query_kwargs)
 
         # extract data to train linear regression model
-        x = ensure_samples_features(self.kdtree_.data[inds])
-        y = ensure_samples_features(self.y_.values[inds])
+        x = ensure_samples_features(self.kdtree_.data[inds-1])
+        y = ensure_samples_features(self.y_.values[inds-1])
```

Comment:
According to this: https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.KDTree.query.html

query should be producing zero based indexes already so perhaps something else is off here.